src/labels/CustomLabels.labels

```diff
@@ -1272,6 +1272,22 @@ If you’re not sure how to resolve these errors, post a message in the Nonprofi
         <shortDescription>Warning</shortDescription>
         <value>Warning</value>
     </labels>
+    <labels>
+        <fullName>RD_ErrorAddDonationPermissionDenied</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>You do not have permission to create a donation record</shortDescription>
+        <value>You do not have permission to create a donation record. Please contact your system administrator for more information.</value>
```

Comment:
@sam-knox - If you get a chance take a look at the wording of this "permission denied" message.  Pretty straightforward but maybe there is something better.